example_configs/nextcloud_oidc_authelia.md

```diff
@@ -0,0 +1,81 @@
+## Authelia
+
+Set up Authelia according to it's [documentation](https://www.authelia.com/overview/prologue/introduction/), including the [OpenID Connect](https://www.authelia.com/configuration/identity-providers/open-id-connect/) and [Nextcloud instructions](https://www.authelia.com/integration/openid-connect/nextcloud/).
+
+## LLDAP
+
+Set up [LLDAP (Light Lightweight Directory Access Protocol](https://github.com/nitnelave/lldap) according to its standard documentation. 
```

Comment:
I think there's no need to spell out LLDAP :D

Reply:
Oversharing from when I first wrote it as a gist :)